packages/veritone-react-common/src/components/DataTable/MenuColumn.js

```diff
@@ -33,6 +34,7 @@ export default class MenuColumn extends React.Component {
     onSelectItem: noop,
     protectedActions: ['delete'],
     transformLabel: l => l,
+    transformActions: a => a,
```

Comment:
seeing this error in the storybook:

![image](https://user-images.githubusercontent.com/693493/51574854-7cbf8580-1e64-11e9-8428-8b7c4e25844f.png)

Reply:
made a recent commit to fix this issue (used omit)